website/docs/d/user.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_user"
+description: |-
+Provides a way to retrieve Sumo Logic user details (id, email, etc) for a user managed by another terraform stack.
+---
+
+# sumologic_role
+
+Provides a way to retrieve Sumo Logic user details (id, email, etc) for a user
+managed by another terraform stack.
```

Comment:
"managed by another terraform stack" -> "managed outside of terraform"

Reply:
+1
Also, looks like the docs for `role` and `collector` data sources also say `"managed by another terraform stack"`. Maybe update those as well while you're at it? cc @troy-q-li